src/Common/MessageBroker/Event/MessagesFlushed.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gaming\Common\MessageBroker\Event;
+
+final class MessagesFlushed
+{
+    public function __construct(
+        public readonly int $numberOfSentMessages
```

Comment:
Add metadata as it's done for the other events. Publishers can then provide contextual information.